entry/src/main/js/default/pages/node_modules/hmos-neumorphism/avatar/avatar.hml

```diff
@@ -0,0 +1,9 @@
+<div class="neumorph_container" style="width: {{width}}; height: {{height}}; border-radius: {{border}};">
```

Comment:
node_modules dont commit, should be generated automatically when dependency is added at compile time